src/strategies/python.ts

```diff
@@ -116,9 +116,9 @@ export class Python extends BaseStrategy {
         );
     }
 
-    // There should be only one version.py, but foreach in case that is incorrect
-    const versionPyFilesSearch = this.github.findFilesByFilenameAndRef(
-      'version.py',
+    // There should be only one *version.py, but foreach in case that is incorrect
+    const versionPyFilesSearch = this.github.findFilesByGlobAndRef(
+      '**/*version.py',
```

Comment:
I feel like this would work for your use case, but could also find unexpected other files to replace. Could there possibly be another submodule that has a version.py file that we wouldn't want to touch?